### PR TITLE
snapp txn tool protocol state predicate

### PR DESCRIPTION
**Pull request by A-Manning**

### CODE REVIEW

File: src/app/snapp_test_transaction/snapp_test_transaction.ml [hunk 1]
```diff
@@ -196,26 +196,39 @@ let generate_snapp_txn (keypair : Signature_lib.Keypair.t) (ledger : Ledger.t) =
...
   in
   printf "Snapp transaction yojson: %s\n\n%!"
     (Parties.to_yojson parties |> Yojson.Safe.to_string) ;
   printf "(Snapp transaction graphQL input %s\n\n%!"
     (graphql_snapp_command parties) ;
-  let consensus_constants =
-    Consensus.Constants.create ~constraint_constants
-      ~protocol_constants:Genesis_constants.compiled.protocol
-  in
+  Core.printf "Updated accounts\n" ;
```
Comment (psteckler) [inline src/app/snapp_test_transaction/snapp_test_transaction.ml:226]: `Core_kernel` is opened above, don't need to quality `printf` here

File: src/app/snapp_test_transaction/snapp_test_transaction.ml [hunk 2]
```diff
@@ -196,26 +196,39 @@ let generate_snapp_txn (keypair : Signature_lib.Keypair.t) (ledger : Ledger.t) =
...
+  List.iter (Ledger.to_list ledger) ~f:(fun acc ->
+      Core.printf "Account: %s\n%!"
```
Comment (psteckler) [inline src/app/snapp_test_transaction/snapp_test_transaction.ml:228]: ditto

**Review by psteckler — APPROVED** [review]
couple of nits, otherwise good

File: src/app/snapp_test_transaction/snapp_test_transaction.ml (hunk 1 above)
Comment (A-Manning) [reply src/app/snapp_test_transaction/snapp_test_transaction.ml]: Done!

File: src/app/snapp_test_transaction/snapp_test_transaction.ml (hunk 2 above)
Comment (A-Manning) [reply src/app/snapp_test_transaction/snapp_test_transaction.ml]: Done!

**Pull request merged**